lib/aws/rails/action_mailbox/sns_message_verifier.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'aws-sdk-sns'
+
+module Aws
+  module Rails
+    module ActionMailbox
+      # @api private
+      class SnsMessageVerifier
+        class << self
+          def client
+            @client ||= Aws::SNS::MessageVerifier.new
```

Comment:
nit: This isn't really a client, maybe just @verifier?

Reply:
true, I wasn't thinking too hard about naming tbh. I'll change